state/protocol/badger/state.go

```diff
@@ -461,6 +474,15 @@ func (state *State) bootstrapSporkInfo(root protocol.Snapshot) func(*badger.Txn)
 			return fmt.Errorf("could not insert spork ID: %w", err)
 		}
 
+		sporkRootBlockHeight, err := params.SporkRootBlockHeight()
+		if err != nil {
+			return fmt.Errorf("could not get spork root block height: %w", err)
+		}
+		err = operation.InsertSporkRootBlockHeight(sporkRootBlockHeight)(tx)
+		if err != nil {
+			return fmt.Errorf("could not insert spork root block height: %w", err)
+		}
+
```

Comment:
## ⚠️ 
I think we are missing the `EpochCommitSafetyThreshold`

Reply:
we insert it in [bootstrapSporkInfo](https://github.com/onflow/flow-go/blob/yurii/6443-sealing-segment-service-event-support/state/protocol/badger/state.go#L497)